docs/migrating-from-knative-build.md

```diff
@@ -1,69 +1,57 @@
-
-* Because of how steps are serialized without relying on init containers, steps
-  should specify a `command` instead of
-  [`entrypoint`](https://github.com/tektoncd/pipeline/blob/master/docs/container-contract.md#entrypoint)
-  and `args`. If the image's `entrypoint` isn't defined, Tekton will attempt
-  to determine the image's entrypoint. This field can be a list of strings,
-  so instead of specifying `args: ['foo', 'bar']` and assuming the image's
-  entrypoint (e.g., `/bin/entrypoint`) as before, you can specify
-  `command: ['/bin/entrypoint', 'foo, bar']`.
-
-## Example
-
-### BuildTemplate -> Task
-
-Given this example BuildTemplate which runs `go test`*:
+# Migrating from Knative Build
+
+Tekton Pipelines is the technological successor to [Knative Build](https://github.com/knative/build). Tekton
+entities are based on Knative Build's entities but provide additional flexibility and reusability. This page
+explains how to convert your Knative Build entities to Tekton entities of equivalent functionality.
+
+The table below lists the mapping between the old and new entities:
+
+| **Knative Build Entity** | **Tekton Entity**  |
+|--------------------------|--------------------|
+| `Build`                  | `TaskRun`          |
+| `BuildTemplate`          | `Task`             |
+| `ClusterBuildTemplate`   | `ClusterTask`      |
+
+## Conversion guidelines
+
+Follow the guidelines below when converting your Knative Build entities to Tekton entities:
+
+* All `steps` must specify a `name` value.
+
+* `BuildTemplate` [parameters](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#parameters)
+  now reside inside the `input.params` field of a `Task`. In a related change, parameter placeholders, such as `${FOO}`,
+  now follow the format `$(input.parameters.FOO)`. For more information, see [Using variable substitution](tasks.md#using-variable-substitution).
+
+* `Tasks` that ingest inputs from resources must now explicitly specify [`input.resources`](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#input-resources).
+  `BuildTemplates` did not explicitly specify input resources and just assumed those resources were always available.
+
+* Input resource data is no longer cloned into the `/workspace` directory. Instead, Tekton clones the data into a subdirectory
+  of your choice within the `/workspace` directory. You must specify this subdirectory using the `name` field in your input
+  resource definition. For example, if you specify a `git` resource named `foo`, Tekton clones the data into `/workspace/foo`.
+  Due to this change, we highly recommend that you set `workingDir: /workspace/foo` in the affected `steps` within your `Task`.
+  For more information, see [Controlling where resources are mounted](resources.md#controlling-where-resources-are-mounted).
+
+* `TaskRuns` which specify a `PipelineResource` as the value of the `input.resources` field of the invoked `Task`
```

Comment:
same as above, s/input.resources/resources.inputs/

Reply:
Not updating this since we're deleting the doc.